lua/supermaven-nvim/util.lua

```diff
@@ -161,8 +161,15 @@ function M.ends_with(str, suffix)
 end
 
 function M.get_utf8_length(str)
-  local byte_array = { str.byte(str, 1, -1) }
-  return #byte_array
+    local ALLOWED_LENGTH = 8000 -- maximum length .byte function may take in LuaJIT (used by Neovim)
```

Comment:
i replaced this with vim.fn.strchars(str) and it seems to work, is counting chars the intended thing here?